ProjFS.Mac/PrjFSKextTests/KauthHandlerTests.mm

```diff
@@ -1,11 +1,22 @@
 #import <XCTest/XCTest.h>
 #include "../PrjFSKext/KauthHandlerTestable.hpp"
+#include <sys/vnode.h>
 
 @interface KauthHandlerTests : XCTestCase
 @end
 
 @implementation KauthHandlerTests
 
+//Mock for TryReadVNodeFileFlags
+static uint32_t mockFileFlag;
+static bool mockReturnBool;
+bool TryReadVNodeFileFlags(vnode_t vn, vfs_context_t context, uint32_t* flags);
```

Comment:
If you need this extra prototype to silence warnings, there's some other problem - you've got a declaration in the header, so this one shouldn't be needed.